components/ide-proxy/conf/Caddyfile

```diff
@@ -1,6 +1,11 @@
 {
 	auto_https off
 	admin off
+	servers {
+		protocol {
+			allow_h2c
```

Comment:
I can find caddy doc about server `allow_h2c` but still don't understand it and why we need it, could you explain? 🙏

Reply:
maybe add a commnet here